pkg/handler-launcher-com/cmd/v1/cmd.proto

```diff
@@ -18,6 +18,7 @@ service Cmd {
   rpc CancelVirtualMachineMigration(VMIRequest) returns (Response) {}
   rpc SignalTargetPodCleanup(VMIRequest) returns (Response) {}
   rpc FinalizeVirtualMachineMigration(VMIRequest) returns (Response) {}
+  rpc HotplugHostDevices(VMIRequest) returns (Response) {}
```

Comment:
Can you please explain how backward compatibility is supported here?
I mean new handler with old already running virt-launchers?
Do we assume that since the `HotplugHostDevices` is part of the migration finalize operation, it will always happen on a new virt-launcher?

Reply:
When virt-handler is upgraded, it will start sending `HotplugHostDevices` commands as part of the reconcile loop.
The already exist virt-launcher pods does not support this command and raise the following error:
```
failed to hot-plug SR-IOV interfaces: unknown error encountered sending command HotplugHostDevices: rpc error: code = Unimplemented desc = unknown method HotplugHostDevices for service
```
But there is no regression, virt-handler will not re-enqueue the requests and virt-handler stays "healthy".

virt-handler sends `HotplugHostDevices` command as part of the reconcile loop (on the regular VM update flow)
instead of expecting virt-launcher to do the hotplug when sending `FinalizeVirtualMachineMigration` only on post migration.